### DISCUSSION TITLE
Karavan Source Code Build Issue

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1198] [Q&A] [answered]
@davsclaus I am getting the source code Issues while building the karavan source in my local machine.

![image](https://github.com/apache/camel-karavan/assets/14121796/f8fcf5a7-5bb1-4f31-b070-25890de0da86)
![image](https://github.com/apache/camel-karavan/assets/14121796/e20af398-e46e-4f0f-bb1c-2f9387bbdd8d)
please help me out from this build issues?????

**Comment by davsclaus** [comment]
I am not active working on karavan - its @mgubaidullin and other community users whom build and work.

I am sure you have seen the docs how to build. But I can't help why you have a build error
Also to be sure the source you build is the latest from `main` branch.

**Comment by kondetiramanakumar** [comment]
@mgubaidullin  please help me out from the above screen shot errors related to the build Issues.

**Comment by kondetiramanakumar** [comment]
@davsclaus  yes you are correct I have seen the docs. But still some of the packages are missing in the sources . FYI I already took the latest sources from my main Branch.

**Comment by kondetiramanakumar** [comment]
@mgubaidullin  import {GetDefinition, RestDefinition} from "karavan-core/lib/model/CamelDefinition" this package itself is not available in the karavan-core sources.

**Reply by mgubaidullin** [reply, marked as answer]
you need to install karavan-core first
